Dockerfiles/agent/Dockerfile

```diff
@@ -34,8 +40,8 @@ RUN find / -name 'datadog-agent*_amd64.deb' ! -name 'datadog-agent-dbg*_amd64.de
     usr/lib/x86_64-linux-gnu/libsystemd.so.0.21.0 \
     # self-test certificates that are detected (false positive) as private keys
     opt/datadog-agent/embedded/lib/python2.7/site-packages/Cryptodome/SelfTest \
- && if [ "$PYTHON_VERSION" = "2" ]; then rm -rf opt/datadog-agent/embedded/lib/python3.*; fi \
- && if [ "$PYTHON_VERSION" = "3" ]; then rm -rf opt/datadog-agent/embedded/lib/python2.*; fi \
+ && if [ "$PYTHON_VERSION" = "2" ]; then rm -rf opt/datadog-agent/embedded/lib/python3.* || true; fi \
```

Comment:
Indeed it's not really necessary, I added it to the line below because A7 would not be shipping `opt/datadog-agent/embedded/lib/python2.*` yet we'd still be specifying `PYTHON_VERSION="3"` and didn't want to break the build there. And added the same above in case we wanted to use the same docker image for an old pre `6.14.x` artifact that might not include `opt/datadog-agent/embedded/lib/python3.*`.